pkg/volume/rbd/rbd_util.go

```diff
@@ -269,7 +269,18 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) (string, error) {
 		if err != nil {
 			return "", fmt.Errorf("error: %v, rbd output: %v", err, rbdOutput)
 		}
-		if found {
+
+		// If accessModes contain ReadOnlyMany, we don't need check rbd status of being used.
+		needValidFound := true
+		if b.accessModes != nil {
+			for _, v := range b.accessModes {
+				if v == v1.ReadOnlyMany {
```

Comment:
Shall we use `v != v1.ReadWriteOnce` instead to make better clarification.
Although `ReadWriteMany` is not allowed for RBD volumes.

Reply:
That't right. I will correct it.